src/js/utils/Pipeline.js

```diff
@@ -0,0 +1,17 @@
+import { isCallable } from "./ValidatorUtil";
+
+export default class Pipeline {
```

Comment:
➡️ Please add unit tests to verify that `Pipeline` works as expected with both other callables as simple functions.

Examples:
```js
expect(new Pipleline(duplicate,square).call(1)).toEqual(square(duplicate(1)))

expect(new Pipleline(duplicate, new Pipleline(square,duplicate)).call(1)).toEqual(duplicate(square(duplicate(1))))
```